src/auto-complete.jsx

```diff
@@ -26,7 +27,7 @@ const AutoComplete = React.createClass({
     /**
```

Comment:
@mbrookes Good catch here is the relevent issue https://github.com/callemall/material-ui/issues/3140.